app/templates/common/base_main.html

```diff
@@ -115,6 +115,143 @@ <h6>The user agrees not to:</h6>
+           your first visit. These can always be re-consulted anytime by clicking on the "Terms of Use" link at the top right 
+           of the portal web page.</p>
+           <img src="/static/img/Terms_of_use.png"alt="Portal_access_from_conp.ca" width="700px"> 
+           <p>Upon agreement to the "Terms of Use", you will be able to filter through the various datasets or pipelines 
+           available on the portal.</p>
+        </ul>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-primary" id="btnAcceptTut1">OK</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="modal fade" id="tutModal2" tabindex="-1" role="dialog" data-backdrop="static" data-keyboard="false"
+  aria-labelledby="modalCenterTitle" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title w-100 text-center" id="modalLongTitle">Help</h4>
+      </div>
+      <div class="modal-body">
+        <ul>
+            <h4>How can I find and download datasets?</h4>
+            <h5>How to find datasets of interest?</h5>
+            <p>Datasets are listed in the Data search page of the portal. This search page 
+            lists all datasets available through the CONP with some basic information about 
+            the dataset. Results can be filtered or sorted easily.</p>
+            <img src="/static/img/Data_search_page.png"alt="Portal_access_from_conp.ca" width="700px">
+            <p>Edit the code in the window to the left, and click "Run" to view the result.</p>
+            <p>Clicking on a dataset row in the Data search page will bring the user to a more 
+            detailed dataset page that includes more description information pertinent to the 
+            dataset as well as download instructions to follow using DataLad.</p>
+            <img src="/static/img/Dataset_detailed_page.png"alt="Portal_access_from_conp.ca" width="700px">
+            <p>Note that some datasets may require a third-party account. Instructions on how to 
+            obtain an account for those datasets will be displayed on the dataset page if that is a requirement.</p>
+            <p>Below are two dataset examples, one that requires a third-party account (PREVENT-AD 
+            open data) and one that does not require any account to download it (BigBrain dataset).</p>
+            <img src="/static/img/Dataset_accounts.png"alt="Portal_access_from_conp.ca" width="700px">
+            <h5>How to download the data?</h5>
+            <p>Dataset download is available through DataLad. Each dataset page contains a detailed 
+            ‘Dataset Download Instructions’ section that lay out the different steps to perform in order 
+            to download the dataset.</p>
+            <img src="/static/img/Dataset_download_instructions.png"alt="Portal_access_from_conp.ca" width="700px">
```

Comment:
This part should not be in the data search page help section. I would instead modify it a little as follows and have that shown in the dataset specific page instead.

```suggestion
            <h4>Dataset Page</h4>
            <p>This page provides more detailed information regarding a dataset. This includes contact information, dataset version, descriptions, licenses, related publication and so on.</p>
            <p>Dataset download is available through DataLad. Each dataset page contains a detailed 
            ‘Dataset Download Instructions’ section that lay out the different steps to perform in order 
            to download the dataset.</p>
            <img src="/static/img/Dataset_download_instructions.png"alt="Portal_access_from_conp.ca" width="700px">
            <p>Finally, the dataset README file can be viewed at the end of the page.</p>
```